app/crunch.sh

```diff
@@ -13,15 +13,15 @@ mkdir -p intermediate
 
 # apply filter, merge with user experience data, aggregate to bins
 # and create z13-z14 tiles for raw data
-./src/index.js $1 $2.json
+node ./src/index.js $1 $2.json
```

Comment:
Any particular reason for this change? On some (debian) systems, node's binary is not called node, but *nodejs* by default. That's why I used `#!/usr/bin/env node` on executables, like here: https://github.com/GFDRR/osm-analytics-cruncher/blob/ac9fcacb826d8a1483c206fca5783128b416e3e6/app/src/index.js#L1